src/eager/readonly_tensor.rs

```diff
@@ -32,6 +36,38 @@ impl<T: TensorType> Deref for ReadonlyTensor<T> {
     }
 }
 
+impl<T: TensorType> Debug for ReadonlyTensor<T> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        let mut counter: i64 = match std::env::var("TF_RUST_DISPLAY_MAX") {
+            Ok(e) => e.parse().unwrap_or(-1),
+            Err(_) => -1,
+        };
+        let dtype = self.data_type().to_string().to_lowercase();
```

Comment:
There's no need to convert it to lowercase.  The case will match the cases of the DataType enum.